lib/api/slack.ex

```diff
@@ -27,20 +28,19 @@ defmodule Aprb.Api.Slack do
         raise("Unauthorized")
       end
 
-      subscriber = case Repo.get_by(Subscriber, user_id: params[:user_id]) do
+      subscriber = case Repo.get_by(Subscriber, channel_id: params[:channel_id]) do
```

Comment:
I kind of liked the idea of subscriber since it could be a user or channel subscribing. But you said DMs are they own channel right? So maybe I can drop `user_id` from there. Right now we store the first subscriber's `user_id` and we never update which I agree sounds weird.